src/channel-slider/channel-slider.js

```diff
@@ -140,6 +147,13 @@ export default class ChannelSlider extends NudeElement {
 		},
 		channelSpec: {
 			get () {
+				if (this.channel === "alpha") {
+					return {
+						name: "Alpha",
+						range: [0, 1],
```

Comment:
Ideally we'd want to expose a 0-100% percentage to users

Reply:
Then, I think we can get rid of `range`; the default `[0, 100]` will be used. But that means we need to adjust the value on every change, right?